---
title: 'txtools: an R package facilitating analysis of RNA modifications, structures, and interactions'
collection: articles
permalink: /publication/2023-08-26-txtools
excerpt: "In this preprint we present txtools, an R package that enables the processing, analysis, and visualization of RNA-seq data at the nucleotide-level resolution, seamlessly integrating alignments to the genome with a transcriptomic representation.' <br/><br/><img src='man/figures/logo.png' align='right' height='200'><br/>"
date: 2023-08-26
venue: 'bioRxiv'
paperurl: 'https://www.biorxiv.org/content/10.1101/2023.08.24.554738' 
citation: 'Garcia-Campos, M. A., & Schwartz, S. (2023). txtools: an R package facilitating analysis of RNA modifications, structures, and interactions. bioRxiv, 2023-08.'
---

Abstract <a href=''><img src='man/figures/logo.png' align="right" height="200" /></a>
==========

We present txtools, an R package that enables the processing, analysis, and visualization of RNA-seq data at the nucleotide-level resolution, seamlessly integrating alignments to the genome with transcriptomic representation. txtools’ main inputs are BAM files and a transcriptome annotation, and the main output is a table, capturing mismatches, deletions, and the number of reads beginning and ending at each nucleotide in the transcriptomic space. txtools further facilitates downstream visualization and analyses. We showcase, using examples from the epitranscriptomic field, how a few calls to txtools functions can yield insightful and ready-to-publish results. txtools is of broad utility also in the context of structural mapping and RNA:protein interaction mapping. By providing a simple and intuitive framework, we believe that txtools will be a useful and convenient tool and pave the path for future discovery. txtools is available for installation from its GitHub repository at https://github.com/AngelCampos/txtools.

<dl>
	<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
	<div data-badge-details="right" data-badge-type="medium-donut" data-doi="https://doi.org/10.1101/2023.08.24.554738" class="altmetric-embed"></div>
</dl>

<!---
<dl>
	<script type="text/javascript" src="//cdn.plu.mx/widget-details.js"></script>
	<a href="https://plu.mx/plum/a/?doi=10.1101%2F571679" class="plumx-details plum-bigben-theme" data-site="plum" data-hide-when-empty="true">Deciphering the ‘m6A code’ via quantitative profiling of m6A at single-nucleotide resolution</a>
</dl>
-->
